doc/input-output-reference/src/hvac-template-objects/group-hvac-templates.tex

```diff
@@ -2456,41 +2448,41 @@ \subsubsection{Inputs}\label{inputs-2016-06-16-1620}
-  autosize;                !- Baseboard Heating Capacity {W}
-  SystemSupplyAirTemperature, !- Zone Cooling Design Supply Air . . . Input Method
-  ,            !- Zone Cooling Design Supply Air Temperature {C]
-  ,            !- Zone Cooling Design Supply Air Temperature Difference {deltaC]
-  SupplyAirTemperature, !- Zone Heating Design Supply Air Temperature Input Method
-  50.0,        !- Zone Heating Design Supply Air Temperature {C]
-  ,            !- Zone Heating Design Supply Air Temperature Difference {deltaC]
-  ,            !- Zone PIU Fan Schedule Name
-  ,            !- Design Specification Outdoor Air Object Name
-  ;            !- Design Specification Zone Air Distribution Object Name
+  SPACE1-1,                   !- Zone Name
+  VAV Sys 1,                  !- Template VAV System Name
+  All Zones,                  !- Template Thermostat Name
+  autosize,                   !- Primary Supply Air Maximum Flow Rate {m3/s}
+  ,                           !- Zone Heating Sizing Factor
+  ,                           !- Zone Cooling Sizing Factor
+  autosize,                   !- Primary Supply Air Minimum Flow Fraction
+  autosize,                   !- Secondary Supply Air Maximum Flow Rate {m3/s}
+  Series,                     !- Flow Type
+  autosize,                   !- Parallel Fan On Flow Fraction
+  flow/person,                !- Outdoor Air Method
+  0.00944,                    !- Outdoor Air Flow Rate per Person {m3/s}
+  0.0,                        !- Outdoor Air Flow Rate per Zone Floor Area {m3/s-m2}
+  0.0,                        !- Outdoor Air Flow Rate per Zone {m3/s}
+  HotWater,                   !- Reheat Coil Type
+  ,                           !- Reheat Coil Availability Schedule Name
+  0.7,                        !- Fan Total Efficiency
+  1000,                       !- Fan Delta Pressure {Pa}
+  0.9,                        !- Fan Motor Efficiency
+  ,                           !- Supply Plenum Name
+  ,                           !- Return Plenum Name
+  None,                       !- Baseboard Heating Type
+  ,                           !- Baseboard Heating Availability Schedule Name
+  autosize;                   !- Baseboard Heating Capacity {W}
+
+SystemSupplyAirTemperature,   !- Zone Cooling Design Supply Air . . . Input Method
+  ,                           !- Zone Cooling Design Supply Air Temperature {C]
+  ,                           !- Zone Cooling Design Supply Air Temperature Difference {deltaC]
+  SupplyAirTemperature,       !- Zone Heating Design Supply Air Temperature Input Method
+  50.0,                       !- Zone Heating Design Supply Air Temperature {C]
+  ,                           !- Zone Heating Design Supply Air Temperature Difference {deltaC]
+  ,                           !- Zone PIU Fan Schedule Name
+  ,                           !- Design Specification Outdoor Air Object Name
+  ;                           !- Design Specification Zone Air Distribution Object Name
```

Comment:
It would be nice not to have extra trailing fields if they are not part of minfields but that is probably another pass.